src/sdk/namespace_fs.js

```diff
@@ -2930,9 +2971,9 @@ class NamespaceFS {
                     const bucket_tmp_dir_path = this.get_bucket_tmpdir_full_path();
                     if (this._is_versioning_enabled() || suspended_and_latest_is_not_null) {
                         await native_fs_utils._make_path_dirs(versioned_path, fs_context);
-                        await native_fs_utils.safe_move(fs_context, latest_ver_path, versioned_path, latest_ver_info,
-                            gpfs_options && gpfs_options.delete_version, bucket_tmp_dir_path);
-                        if (suspended_and_latest_is_not_null) {
+                         await native_fs_utils.safe_move_posix(fs_context, latest_ver_path, versioned_path, latest_ver_info,
```

Comment:
we said we need that also on _delete_latest_version, right? I couldn't find it

Reply:
right we need it whenever we move to .version

Reply:
looks like the move in _delete_latest_version is already safe_move_posix